docs/changes.xml

```diff
@@ -31,6 +31,12 @@ NGINX Unit updated to 1.27.0.
          date="" time=""
          packager="Andrei Belov &lt;defan@nginx.com&gt;">
 
+<change type="feature">
+<para>
+supporting variables in the "location" option of the "return" action.
```

Comment:
Previous changelog entries had it like this: `variables support in the "XXXYYY" option...`

Reply:
That's what I wrote at first for the "index" patch set, IIRC, but was suggested to write it in this other form.

Which way do you prefer?

Cc: @hongzhidao
Cc: @tippexs

Reply:
Changes applied as suggested.